src/lib.rs

```diff
@@ -32,6 +32,33 @@ pub struct Wiki {
     paths: Vec<PathBuf>,
 }
 
+fn get_http_error_as_html(status: iron::status::Status) -> (iron::mime::Mime, iron::status::Status, &'static str) {
+
+    match status {
+        status::NotFound =>
+            (ContentType::html().0, status, "
+<html>
```

Comment:
Okay since we want to have it hard coded what about the usage of [`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html)?